store/build.gradle

```diff
@@ -11,3 +14,25 @@ kotlin {
     coroutines 'enable'
   }
 }
+
+sqldelight {
+  packageName = "com.jakewharton.sdksearch.store"
+}
+
+compileKotlinCommon.dependsOn("generateSqlDelightInterface")
```

Comment:
Should the plugin do this?

Reply:
...yes. Right now it tries to add it to the `compileKotlin` task but we should be doing for `compileKotlinCommon` as well